src/libcore/default.rs

```diff
@@ -47,6 +47,25 @@
 /// }
 /// ```
 ///
+/// In case you feel that `Default::default()` is repeating the word "default"
+/// too many times, you can import a free [`default()`] function instead:
+///
+/// ```
+/// #![feature(default_free_fn)]
+///
+/// use std::default::default;
+///
+/// #[derive(Default)]
+/// struct SomeOptions {
+///     foo: i32,
+///     bar: f32,
+/// }
+///
+/// fn main() {
+///     let options = SomeOptions { foo: 42, ..default() };
+/// }
+/// ```
```

Comment:
Please remove. I would prefer not to feature an unstable API so heavily in the doc of an extremely widely used trait.

Reply:
I guess, I got overly eager :)
It totally makes sense not to promote an unstable API.
Removed.